source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -558,6 +558,14 @@ def __init__(self):
             help="enhance response with accessibility equipement details",
         )
 
+        parser_get.add_argument(
+            "_enable_instructions",
+            type=BooleanType(),
+            default=True,
+            hidden=True,
+            help="Enable/Disable the narrative instructions for street network sections",
+        )
+
```

Comment:
Is it possible to add at least a test on this parameter ?

Reply:
It should be used with asgard.... it's almost impossible to test this... :(